src/listener.rs

```diff
@@ -61,8 +63,10 @@ impl EventBusListener {
 fn reader_loop(read_stream: TcpStream, handlers: MessageHandlersByAddress) {
     let mut socket = buffered_reader::Generic::new(&read_stream, Some(4096));
     loop {
+        thread::sleep(Duration::from_millis(100));
```

Comment:
Is this intentional? It looks like test code.

Reply:
Yes it is, in fact, I noticed the loop was generating a lot of error messages ("Would block") and thuis realised that polling the socket continuously could be really CPU intensive, when a single poll every 100ms or so (maybe configurable?) could be enough.

What do you think? Bad idea?

Reply:
I am not too familiar with the `TcpStream` API. I will also take a look.
But this could potentionally mean that there will be high latencies, when the data is already arrived. 
Also if there a lot of messages, it means that there will be 100ms between each message to be processed (it could be that I misunderstand).

Reply:
True.